src/wp-includes/block-supports/duotone.php

```diff
@@ -69,6 +69,27 @@ function wp_tinycolor_bound01( $n, $max ) {
 	return ( $n % $max ) / (float) $max;
 }
 
+/**
+ * Direct port of tinycolor's boundAlpha function to maintain consistency with
+ * how tinycolor works.
+ *
+ * @see https://github.com/bgrins/TinyColor
+ *
+ * @since 5.9.0
+ *
+ * @param  mixed $n   Number of unknown type.
+ * @return float      Value in the range [0,1].
+ */
+function wp_tinycolor_bound_alpha( $n ) {
```

Comment:
This function should be marked as private prefixed with "_". This is internal only and we should look into renaming these color functions ASAP before any beta version of 5.9 comes out. The functions should be named in what they do and not where the inspiration comes from. Now we don't even use tinycolor on the client, we should not use a port of it on the server. We should look into what color functions we need on the server and name them accordingly.
This is an issue that comes from WP 5.8 so it is not a blocker provided we mark it as private with a to-do comment to remove or rename.

Reply:
https://github.com/WordPress/wordpress-develop/pull/1808/commits/33a03551dce87c4da25b52656760c131798b1c9c